changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
+  - app_icons
+  - apple_silicon_security_policy
+  - falconctl_options
+  - falcon_kernel_check
+  - cryptoinfo
+  - cryptsetup_status
+  - parse_plist
+  - parse_json
+  - parse_jsonl
+  - parse_xml
+  - parse_ini
+  - dev_table_tooling
+  - dsim_default_associations
```

Comment:
```suggestion
```

Noah: Let's cut this table. It looks hard to use and I don't know if it's valuable. 

It looks like the data it returns gets dumped to a file? https://github.com/fleetdm/fleet/pull/14613/files#diff-9dbdf950f412a0e7de7d2bad14b91125d73029836891ef76ddc8b881cdb419a3R69